templates/docs/examples/patterns/notifications/caution.html

```diff
@@ -5,8 +5,9 @@
 
 {% block content %}
 <div class="p-notification--caution">
-  <p class="p-notification__response" role="status">
-    <span class="p-notification__status">Blocked:</span>Custom storage configuration is only supported on Ubuntu, CentOS and RHEL.
```

Comment:
We try to have "real" (or at least real-like) content in the examples. It would be nice to keep the text from old notification examples instead of generic lorem ipsum.